modules/react-maplibre/src/utils/set-globals.ts

```diff
@@ -4,7 +4,7 @@ export type GlobalSettings = {
    */
   maxParallelImageRequests?: number;
   /** The map's RTL text plugin. Necessary for supporting the Arabic and Hebrew languages, which are written right-to-left.  */
-  RTLTextPlugin?: string | false;
+  RTLTextPlugin?: {pluginUrl: string; lazy?: boolean};
```

Comment:
Not a blocker, but it shouldn't be hard to allow a string value as well?

Reply:
You are right, reimplemented!